troposphere/static/js/components/requests/ResourceHistoryMaster.jsx

```diff
@@ -1,30 +1,14 @@
 import React from "react";
 
 import ResourceActions from "actions/ResourceActions";
```

Comment:
Matching the Redux Connect interface, the actions could be mapped onto props.actions by subscribe. This way the component is pure and more reusable. This makes testing easier as well, since we could pass a mock action and assert the result.